Bindings/Packages/com.virtualmaker.bindings/Runtime/Bindings.InputActions.cs

```diff
@@ -0,0 +1,190 @@
+#if UNITY_INPUT_SYSTEM
+using System;
+using UnityEngine;
+using UnityEngine.InputSystem;
+
+namespace VirtualMaker.Bindings
+{
+    public partial class Bindings
+    {
+        public void Bind<T>(InputActionReference inputActionReference,
```

Comment:
I think it would make more sense to split these out

OnPerformed
OnStarted
OnCanceled
BindPerformed
BindStarted
BindCanceled

Reply:
sementics, but sure.

Reply:
I don't quite understand the full recommendation here.

Do you want me to rename the parameters or do you want to split out the Method overloads?

I'm fine with renaming, but prefer to keep the single method. That's why the parameters are optional, so we don't have a bunch of redundant method signatures.

Reply:
I refactored it a bit more and co-located the prop and action callbacks